### PR TITLE
Added a calorie burnt prediction model 

### LINKED ISSUE #989: Gssoc 24' Calorie burnt prediction
### Is your feature request related to a problem? Please describe.

The feature request I provided isn't directly related to a specific problem. Instead, it's a set of guidelines for predicting calories burnt outcomes within a project context. If there's a specific problem you're facing related to calorie tracking in a project, feel free to share, and I can tailor my response to address that particular issue.

### Describe the solution you'd like.

I might implement ml strategies and algorithms to predict calories burnt in strenuous activities within the project.

### Describe alternatives you've considered.

Another alternative machine learning (ML) model that could be considered is a deep learning model specifically trained on data of various people.

### Add any other context or screenshots about the feature request here.

_No response_

### PR DESCRIPTION
**Pull request by Seersha9802**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

yes I have

### Description

Adding this project to Basic section of Data Science and Machine Learning folder with main folder name as "Calorie Burnt Prediction". The project is done in a Jupyter Notebook and python libraries like pandas, numpy, matplotlib and seaborn are used for visualizations and XGBoost for prediction.

Fixes #989 

### Checklist

- [yes] I've read the contribution guidelines.
- [yes] I've checked the issue list before deciding what to submit.
- [yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

This is the related PR and the issue fixed is https://github.com/Kushal997-das/Project-Guidance/issues/989

### CODE REVIEW

File: Machine Learning and Data Science/Basic/Calorie Burnt Prediction/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Calorie Burnt Prediction 
+The following project deals with datasets of users and the calories burnt and prediction is made and accuracy is achieved.
```
Comment (Kushal997-das) [inline Machine Learning and Data Science/Basic/Calorie Burnt Prediction/README.md:2]: Add images and descriptions in more detail.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Seersha9802 Things you need to change :
1. Add good images in `readme.md` file also add good description about your project like how to use? what exactly the project is about everything you need to mention.
2. Create a folder called `dataset` add your `.csv` file over there.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Seersha9802 Kindly look into the comment and update accordingly.

**Comment by Seersha9802** [comment]
Yes, I'm working on it.

**Comment by Seersha9802** [comment]
@Kushal997-das  I have made the changes, Kindly go through it .

**Review by Kushal997-das — APPROVED** [review]

**Pull request merged**